src/renderer/store/modules/profile.js

```diff
@@ -16,6 +16,22 @@ export default new BaseModule(ProfileModel, {
       return wallets.reduce((total, wallet) => {
         return total + wallet.balance
       }, 0)
+    },
+    balanceWithLedger: (state, _, __, rootGetters) => id => {
+      let wallets = rootGetters['wallet/byProfileId'](id)
+
+      // Only include the wallets of the Ledger that are on the same network than the profile
+      const profile = rootGetters['profile/byId'](id)
+      if (profile.networkId === rootGetters['session/network'].id) {
```

Comment:
Might it be simpler to just do this based on current profile only? Maybe move it to the session store instead? Ledger only loads wallets for the current profile, not all profiles. It reloads wallets when switching profile

Reply:
Yes, this should be changed, but I'm not sure if I'd keep it or not, because I was thinking on a different approach: on the profiles page, show the balance of all the profile networks that are available. What do you think @alexbarnsley?

Reply:
I thought it already showed the balance of each network already? 🤔: 

![image](https://user-images.githubusercontent.com/8069294/49657320-55893200-fa37-11e8-9cdb-92b51a2cfccd.png)

Reply:
Saying that, is this method used when working out those totals when including the ledger? If so, it makes sense to keep this method

Reply:
It is included to display the total of all the wallets on the wallets page and, on each profile of the profiles page (the total of all profiles is calculated differently).

About the total of each network, I'm talking about having 2 Ledger wallets on mainnet with 200ARK, and 3 Ledger wallets with 3000DARK. If you are using a profile that connects to mainnet, you would see 200ARK, but not the other 3000DARK. If you change to a profile on devnet, it would be the reverse.
Ideally, the wallet should display both balances.

Reply:
Yeh we'll get this merged. However, I don't think it's a good idea to load all ledgers for all networks. Some people have a lot of ledger wallets which could result in a very slow load of wallets when plugged in. It's unfortunately already slow because of having to do an API request for each wallet to determine if it's a cold wallet or not 🤔

Reply:
Yes, that's my main concern, but the `Synchronizer` could be improved further and, after that, keeping the wallets in sync would be much more efficient, although the problem would be on the initial load. I'll try to have that changed next week.